doc/tutorial/c_tutorial/c_tutorial.rst

```diff
@@ -0,0 +1,487 @@
+
+.. tip:: This is a tip on DDlog syntax concerning comments in the code.
+
+    DDlog supports C-style comments as you can see in the example above.
+
+As we have changed the DDlog program we need to recompile it.
+
+.. code-block::
+
+    $ ddlog -i t1_reachability_monitor.dl
+    $ cd t1_reachability_monitor_ddlog/ && cargo build --release
+    $ ./target/release/t1_reachability_monitor_cli
+    <clip>
+    >> dump ConnectedNodes;
+    ConnectedNodes{.src = "Los Angeles", .dst = "Salt Lake City"}
+    ConnectedNodes{.src = "Los Angeles", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Salt Lake City"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Salt Lake City", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Salt Lake City", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Salt Lake City"}
+
+Perfect!
+Now it looks exactly as we expected!
+Rigth before we jump to what this tutorial promised let's look at one more example that demonstrates incremental nature of DDlog.
+
+.. code-block::
+
+    $ ./target/release/t1_reachability_monitor_cli
+    >> start;
+    >> insert Links("Menlo Park", "Santa Barbara", true);
+    >> insert Links("Menlo Park", "Salt Lake City", true);
+    >> insert Links("Santa Barbara", "Los Angeles", true);
+    >> insert Links("Los Angeles", "Santa Barbara", true);
+    >> insert Links("Los Angeles", "Salt Lake City", true);
+    >> insert Links("Salt Lake City", "Los Angeles", true);
+    >> commit;
+    >> dump ConnectedNodes;
+    ConnectedNodes{.src = "Los Angeles", .dst = "Salt Lake City"}
+    ConnectedNodes{.src = "Los Angeles", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Salt Lake City"}
+    ConnectedNodes{.src = "Menlo Park", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Salt Lake City", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Salt Lake City", .dst = "Santa Barbara"}
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Los Angeles"}
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Salt Lake City"}
+    >> start;
+    >> delete Links("Santa Barbara", "Los Angeles", true);
+    >> commit dump_changes;
+    ConnectedNodes:
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Los Angeles"}: -1
+    ConnectedNodes{.src = "Santa Barbara", .dst = "Salt Lake City"}: -1
+
+DDlog incrementally computed only changes that happened and printed them.
+The "minus one" here is called :code:`weight` and indicates that the respective record was deleted.
+While the benefits of incremental computation ain't noticeable in our small example, they manifest themselves on a large scale and can make a substantial difference.
+
+Now we are finally realdy to start writing some C code!

```

Comment:
why not make the text up to here the first tutorial and this one the second one?
You really have two tutorials: first is how to compile and run the CLI, second is how to integrate into a C program.
No additional work required.